src/google/calendar.ts

```diff
@@ -295,7 +325,7 @@ export function watchForChanges() {
   const watcher = new MutationObserver(onMutation);
 
   watcher.observe(document.body, {
-    attributes: false,
+    attributes: true,
```

Comment:
I didn't notice any slowdowns, likely because we don't handle the vast majority of child-list/attribute changes. That said, I did add the `attributeFilter` to pre-filter anything that is not the `style` attribute. I then added some `console.count` instances to see how often mutation of each type occurred.

When Google Calendar is loaded, before any interaction, I saw 111 child list changes, and 81 attribute changes. Clicking to create a new event yielded an additional 184 child list changes, but only 26 new attribute changes.

Reply:
Great, thanks for looking!